spaces/domains.json

```
@@ -3,6 +3,7 @@
   "vote.karameladao.com": "karameladao.eth",
   "vote.ravethefuture.com": "ravethefuture.eth",
   "governance.cyberestate.io": "cyberrealestate.eth",
+  "gov.meritvault.xyz": "meritvault.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "gov.meritvault.xyz": "meritvault.eth",
```